apps/web/src/pages/Home/HomePage/components/Opportunities/Opportunities.tsx

```diff
@@ -126,24 +128,25 @@ const Opportunities = () => {
               </p>
             </CardFlat>
             <CardFlat
-              color="blue"
+              color="secondary"
               title={intl.formatMessage({
                 defaultMessage: "Indigenous Apprenticeship Program",
                 id: "XR37x0",
                 description:
                   "Heading for the Indigenous Apprenticeship Program on home page",
               })}
-              link={{
-                href: `/${locale}/indigenous-it-apprentice`,
-                external: true,
-                label: intl.formatMessage({
-                  defaultMessage:
-                    "Apply<hidden> to the Indigenous Apprenticeship Program</hidden> now",
-                  description:
-                    "Link text to apply for the Indigenous Apprenticeship Program",
-                  id: "w3Kkk2",
-                }),
-              }}
+              links={[
```

Comment:
Looks like we lost the colour on this button.
![image](https://user-images.githubusercontent.com/8978655/229212836-acb214f1-d984-4049-bb4e-d2051a883f5d.png)
And these too.
![image](https://user-images.githubusercontent.com/8978655/229212949-c30d87e5-324d-4fc4-845c-05c726b8d778.png)

Reply:
Oh, I thought that was part of the colour rename so I fixed it in that commit.

Should be good now after [update flat cards to use generic colour names](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6143/commits/768b6729d2dbabf67a2387e4608d17cf3671101c)